src/content/loaders/powerPizzaPodcastLoader.ts

```diff
@@ -0,0 +1,107 @@
+import { feedLoader } from '@ascorbic/feed-loader';
+import type { Loader, LoaderContext } from 'astro/loaders';
+import { DataEntry, z } from 'astro:content';
+import type { ZodObject, ZodSchema } from 'astro:schema';
+
+interface Link {
+    href: string;
+    description: string;
+}
+
+/**
+* Extract links from "Scontrino" content
+*
+* This parser decorates the default feed object, coming from the feed loader, with the list of links in the "scontrino".
+* It allows to reuse them across the project without errors.
+*/
+function parseLinks(episode: any): Link[] {
+    const links: Link[] = [];
+
+    const SCONTRINO_SEPARATOR = "+++ Scontrino +++";
+
+    let scontrino: string;
+    if (episode.data.description!.includes(SCONTRINO_SEPARATOR)) {
+        scontrino = episode.data.description!.split(SCONTRINO_SEPARATOR)[1];
+    } else {
+        scontrino = episode.data.description!;
+    }
+    const scontrinoLines = scontrino
```

Comment:
string cleaner had already some functions to do this cleanup https://github.com/vikkio88/ppp-notes/pull/3/files#diff-34d7e6a8c80c358c6a3bab3d13abfdd6e5ce6e80a3b0ae438b4da70fc1d1d4f1L1